main.tf

```diff
@@ -40,21 +49,26 @@ resource "aws_subnet" "private" {
   cidr_block = local.private_cidr[count.index]
 
   tags = {
-    Name = "private${count.index}"
+    #Name = "private${count.index}"
```

Comment:
Please don't leave commented-out code blocks. That's considered a bad style.
Please fix it here and in all other parts of code that has similar issue